api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -66,6 +69,9 @@
 	private UserService userService;
 
 	private MessageSourceService messages;
+	
+	@Autowired(required = false)
```

Comment:
Why do you have to add required = false? Please remove it because I don't think it's necessary